vm/src/vm.rs

```diff
@@ -1334,6 +1334,21 @@ impl VirtualMachine {
         reflection: &str,
         unsupported: fn(&VirtualMachine, &PyObject, &PyObject) -> PyResult,
     ) -> PyResult {
+        if rhs.isinstance(&lhs.clone_class()) {
+            let lop = lhs.get_class_attr(reflection);
+            let rop = rhs.get_class_attr(reflection);
+            if let Some(lop) = lop {
+                if let Some(rop) = rop {
+                    if !lop.is(&rop) {
+                        if let Ok(r) = self.call_or_unsupported(rhs, lhs, reflection, |vm, _, _| {
+                            Err(vm.new_exception_empty(vm.ctx.exceptions.exception_type.clone()))
+                        }) {
```

Comment:
I'm pretty sure this can be swapped out to `call_special_method` + `PyArithmeticValue::from_object`, but it's probably fine for now.

Reply:
It must be revised after number protocol. So go for easy way for now.